src/umpire/interface/c_fortran/typesUmpire.h

```diff
@@ -30,11 +24,11 @@
 extern "C" {
 #endif
 
-struct s_UMP_SHROUD_capsule_data {
+struct s_umpire_SHROUD_capsule_data {
```

Comment:
Will these name changes (from `s_UMP_` to `s_umpire_`) cause any externally visible API changes?

Reply:
Nope, these are internal.